packages/cli/src/commands/backend/buildImage.ts

```diff
@@ -34,11 +34,15 @@ export default async (cmd: Command) => {
   const pkgPath = paths.resolveTarget(PKG_PATH);
   const pkg = await fs.readJson(pkgPath);
   const appConfigs = await findAppConfigs();
+  const npmrc = fs.existsSync(paths.resolveTargetRoot('.npmrc'))
```

Comment:
very small nit: prefer `await fs.exists`

Reply:
> fs.exists() is deprecated, but fs.existsSync() is not. The callback parameter to fs.exists() accepts parameters that are inconsistent with other Node.js callbacks. fs.existsSync() does not use a callback.

https://nodejs.org/api/fs.html#fs_fs_existssync_path

Reply:
From the fs-extra package, not fs

Reply:
(We pretty rarely import from fs, given how good fs-extra is)

Reply:
> fs-extra is a superset of fs, inheriting all its methods, so fsExtra.exists is the same as fs.exists. 
https://stackoverflow.com/questions/56488179/what-is-the-difference-between-fsextra-pathexists-and-fsextra-exists

seems like we are using `fs.existsSync`
```
$ git grep fs.exist
.yarn/releases/yarn-1.22.1.js:      // for versions before node v0.8 when there weren't `fs.existsSync`
packages/cli-common/src/paths.ts:    const exists = fs.existsSync(packagePath);
packages/cli-common/src/paths.ts:  const isLocal = fs.existsSync(resolvePath(ownDir, 'src'));
packages/cli/config/jest.js:  if (fs.existsSync('src/setupTests.ts')) {
packages/cli/src/commands/backend/buildImage.ts:  const npmrc = fs.existsSync(paths.resolveTargetRoot('.npmrc'))
plugins/scaffolder-backend/src/scaffolder/stages/prepare/file.test.ts:    expect(fs.existsSync(`${resultDir}/template.yaml`)).toBe(false);
plugins/scaffolder-backend/src/scaffolder/stages/prepare/file.test.ts:    expect(fs.existsSync(`${resultDir}/expected_file.ts`)).toBe(true);
plugins/scaffolder-backend/src/scaffolder/stages/prepare/file.test.ts:    expect(fs.existsSync(`${resultDir}/expected_file.ts`)).toBe(true);
plugins/techdocs-backend/src/helpers.ts:  if (fs.existsSync(repositoryTmpPath)) {
plugins/techdocs-backend/src/techdocs/stages/publish/local.test.ts:    expect(fs.existsSync(resultDir)).toBeTruthy();
plugins/techdocs-backend/src/techdocs/stages/publish/local.test.ts:    expect(fs.existsSync(path.join(resultDir, '/mock-file'))).toBeTruthy();
plugins/techdocs-backend/src/techdocs/stages/publish/local.ts:    if (!fs.existsSync(publishDir)) {
scripts/check-type-dependencies.js:    fs.existsSync(resolvePath(pkg.location, 'dist/index.d.ts'))
```

Reply:
I know they're effectively the same - what I'm after is the promise style and await.

Reply:
Note that the usages are in cli, script, tests. Yeah I know this is in the cli so it's really not a big deal either way.

Reply:
when I put in `await`, TS says there's no effect on the type.
<img width="469" alt="Screenshot 2020-11-30 at 1 42 16 AM" src="https://user-images.githubusercontent.com/1654592/100549328-48f9e600-32ad-11eb-99cb-c0223d173655.png">

Reply:
@feiming Use `fs.pathExists` from fs-extra, it's the non-deprecated async version

Reply:
Aw crap, thanks

Reply:
done